currency-demo/src/components/CurrencyConverter.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <v-card>
+        <v-card-text>
+            <v-text-field
+                v-model="usdValue"
+                label="USD"
+                type="number"
+            ></v-text-field>
+            <v-text-field
+                v-model="inrValue"
+                label="INR"
+                type="number"
+                readonly
+            ></v-text-field>
+        </v-card-text>
+    </v-card>
+</template>
+
+<script>
+export default {
+    name: "CurrencyConverter",
+    props: {
+        value: {
+            type: Number,
+            default: 0,
+        },
+        exchangeRate: {
```

Comment:
why do we need to pass this from parent?. its a constant and belongs to the childso can we handle this in the child only.
only that data needs to be passed from parents which is part of parent. the child relatied stuffs needs to be in the child.

Reply:
It is only for demonstration purpose. In the future, I could make it dynamic to convert currency into any other currency.

I will do the required changes.